interview-questions.md

```diff
@@ -15,54 +15,54 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Remembering back from the notes i took, git is the technology aspect or the (tool). GitHub is a website that receives and stores the code or information. 
 
-  Researched answer:
+  Researched answer: Git is the tool that i mentioned in my answer that can have an unlimited amount of versions of code. GitHub is mainly used for sending repositories and updating them constantly, especially when i am pair programming.
```

Comment:
Excellent! Yea, Git is a software that is installed on computers like any application.